### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by ad-m**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,65 @@
...
+## Requirements
 
-### Go
+* [Docker Machine](https://docs.docker.com/machine/install-machine)
+* [Go tools](https://golang.org/doc/install) (only for installation from sources)
+* HyperOne account & access to *Project*
+
+## Installation
+
+### via Go tools
```
Comment (gregorybrzeski) [inline README.md]: "using Go tools" or just "Go tools"

File: README.md [hunk 2]
```diff
@@ -1,32 +1,65 @@
...
+Download the binary (this example downloads the binary for darwin amd64):
 
 ```shell
-# Download the binary (this example downloads the binary for darwin amd64)
 $ wget https://github.com/hyperonecom/docker-machine-driver-hyperone/releases/download/v0.0.1/docker-machine-driver-hyperone_0.0.1_darwin_amd64.zip
-$ unzip unzip docker-machine-driver-hyperone_0.0.1_darwin_amd64.zip
+$ unzip docker-machine-driver-hyperone_0.0.1_darwin_amd64.zip
+```
+
+Make it executable and copy the binary in a directory accessible with your $PATH:
```
Comment (gregorybrzeski) [inline README.md]: I would suggest "accessible on your $PATH"

File: README.md [hunk 3]
```diff
@@ -67,22 +100,20 @@ Options:
...
-Waiting for SSH to be available...
-Detecting the provisioner...
-Provisioning with debian...
-Copying certs to the local machine directory...
-Copying certs to the remote machine...
-Setting Docker configuration on the remote daemon...
-Checking connection to Docker...
-Docker is up and running!
-To see how to connect your Docker Client to the Docker Engine running on this virtual machine, run: docker-machine env vm
+$ go build
```
Comment (gregorybrzeski) [inline README.md:121]: I would include previous commands as well, as this seems to be an instruction for a newcomer to Go world:

```shell
$ go get ....
$ cd $GOPATH/.....
$ go build
```

Comment (ad-m) [reply README.md:121]: @gregorybrzeski , ```go build``` automatically fetch dependencies. There is no need to use "go get" if the command is executed from the root of repository

Comment (gregorybrzeski) [reply README.md:121]: How do you get the the repository in which you execute the `go build` then ? IMHO that step is missing here.

**Pull request merged**